source/slang/slang-lower-to-ir.cpp

```diff
@@ -9344,6 +9344,11 @@ struct DeclLoweringVisitor : DeclVisitor<DeclLoweringVisitor, LoweredValInfo>
                 switch (requirementVal->getOp())
                 {
                 default:
```

Comment:
I see a non-`default` case below. Is there a reason why the case for associated types should retain the linkage decorations while the default case removes them? Is it possible that the `removeLinkageDecorations()` call should be made before the `switch` rather than inside it?

Reply:
I have not digged into the associatedtype case enough to know if this is what we should do. My understanding is that assosicatedtype here is nothing more than a placeholder, and is not really used in any meaningful way.